lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2380,6 +2390,23 @@ Function *SPIRVToLLVM::transFunction(SPIRVFunction *BF) {
   return F;
 }
 
+Value *SPIRVToLLVM::transAsmINTEL(SPIRVAsmINTEL *BA, Function *F,
```

Comment:
Arguments `F` and `BB` are not used - this might result in a warning, and therefore error in some configurations

Reply:
Done